libminifi/include/utils/BackTrace.h

```diff
@@ -41,20 +42,16 @@ class TraceResolver;
  */
 class BackTrace {
  public:
-  BackTrace() {
-  }
-  BackTrace(const std::string &name)
-      : name_(name) {
+  BackTrace() = default;
+
+  explicit BackTrace(std::string name)
+      : name_(std::move(name)) {
   }
-  BackTrace(BackTrace &&) = default;
-  BackTrace(BackTrace &) = delete;
```

Comment:
Making it copyable in addition to being movable doesn't break the previous promise. We can keep it non-copyable if desired, but I didn't see the point and since both declarations were wrong I decided to rely on the implicit copy and move ctors.

Given the above, do you still want to disable copy or change the description? (I'd just remove the "movable" word.)

Reply:
I am OK with it being copyable, but then the description should be changed.

Reply:
changed description